### PR TITLE
Docs/Update QA for "hide zero balance tokens" feature

### PR DESCRIPTION
**Pull request by tuliomir**
### Acceptance Criteria
- There should be new steps for testing the "Hide zero-balance tokens" on the QA document

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -140,22 +169,22 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
 1. **Register tokens with same name**
     1. Unregister TST token but copy its configuration string and save it.
     1. Click on 'Custom tokens' and then 'Create a new token'.
-    1. Type 'Test token' in the token name, 'Test' in the symbol and 200 in the amount.
+    1. Type 'Test token' in the token name, 'TST' in the symbol and 200 in the amount.
```
Comment (andreabadesso) [inline QA.md:172]: For this step, the symbol don't necessarily needs to be the same as the previous one as the check is on the token name

I'm not agains't this change tho

File: QA.md [hunk 2]
```diff
@@ -140,22 +169,22 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
...
+    1. Click on the first transaction with amount 0.00.
     1. Copy the first input from HTR and from your address (copy the transaction ID and the index).
     1. Click on 'Send tokens'.
     1. Paste the address and 1.00 as the amount. Deselect 'Choose inputs automatically' and paste the transaction ID and index copied before.
     1. Must see an error that the output is already spent.
 
 1. **Create NFT**
     1. Click on 'Custom tokens', then 'Create an NFT'.
-    1. Fill NFT data as 'test data', name and symbol as 'NFT Test' and 'NFTT', and amount as 100. Check only the melt authority checkbox.
+    1. Fill NFT data as `ipfs://test`, name and symbol as 'NFT Test' and 'NFTT', and amount as 100. Check only the melt authority checkbox.
```
Comment (andreabadesso) [inline QA.md:187]: Oh yea, that would work too

But I think we can remove the ipfs:// from the next step that uses it

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: QA.md (hunk 1 above)
Comment (tuliomir) [reply QA.md:172]: If only the name needs to be the same, I would rather revert my change to keep this clear. :+1:

File: QA.md (hunk 2 above)
Comment (tuliomir) [reply QA.md:187]: My intent here was just to keep the data consistent with the validation we make a few steps after.
Keeping both `ipfs://test` or `test data` would suffice.

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**